curves/src/traits/group.rs

```diff
@@ -224,6 +224,9 @@ pub trait AffineCurve:
 
     /// Performs the second half of batch addition in-place.
     fn batch_add_loop_2(a: &mut Self, b: Self, inversion_tmp: &mut Self::BaseField);
+
+    /// Sets the `t` coordinate to the correct value.
+    fn correct_t_coordinate(&mut self);
```

Comment:
why can't we do the correction directly in `batch_add_loop_2`? That way we don't have to expose this method for short weierstrass curves.

Reply:
I assumed we should do it as little as possible since it would incur a multiplication, so I exposed it to be able to call it after running the add loops

Reply:
Hm for TE curves I believe we use the standard MSM, not the one with the batch-affine, so we don't call these methods anyway

Reply:
ah sure, in that case I will just do the correction in `batch_add_loop_2` then :)